driver-pulsar/src/main/java/io/openmessaging/benchmark/driver/pulsar/PulsarBenchmarkDriver.java

```diff
@@ -63,63 +61,61 @@
 
     private PulsarConfig config;
 
-    private ProducerConfiguration producerConfiguration;
 
     private String namespace;
+    private ProducerBuilder<byte[]> producerBuilder;
 
     @Override
     public void initialize(File configurationFile, StatsLogger statsLogger) throws IOException {
         this.config = readConfig(configurationFile);
         log.info("Pulsar driver configuration: {}", writer.writeValueAsString(config));
+        client = PulsarClient.builder().ioThreads(config.client.ioThreads)
+                        .connectionsPerBroker(config.client.connectionsPerBroker).statsInterval(0, TimeUnit.SECONDS)
+                        .serviceUrl(config.client.serviceUrl).build();
 
-        ClientConfiguration clientConfiguration = new ClientConfiguration();
-        clientConfiguration.setIoThreads(config.client.ioThreads);
-        clientConfiguration.setConnectionsPerBroker(config.client.connectionsPerBroker);
 
-        // Disable internal stats since we're already collecting in the framework
-        clientConfiguration.setStatsInterval(0, TimeUnit.SECONDS);
-        log.info("Set Pulsar client configuration: {}", writer.writeValueAsString(clientConfiguration));
+        //log.info("Set Pulsar client configuration: {}", writer.writeValueAsString(client.));
```

Comment:
Remove commented log line. There's currently no good way to get a printout of the builder settings. We'll figure something out to have some proper way to add that in the interface.